utils/context.py

```diff
@@ -229,7 +229,6 @@ def reset(self):
         """Reset existing instances of dynamic class members
 
         """
-        del self.irods_connector
```

Comment:
This should NOT be removed.  Without it, the conditional at the beginning of the `iBridges.login_function()` will not provide a new connector and there will be remnants of the old session remaining.

Reply:
I see now that a previous commit modified the `login_function()` functionality incorrectly.  The `irods_connector` MUST be set initially in `main()` and again in `login_function()` _after_ a new session is requested.  If this is not done, the _new_ session will simply be the _old_ one!

Reply:
Regardless of the effect removing it has on the code, this method is meant to _reset_ the `Context` instance to as close to its original state as possible.  _Not_ deleting the connector violates that concept.